build/test.ps1

```diff
@@ -38,7 +38,17 @@ function Test-One {
     }
 }
 
+function Test-VSCode {
+    Param([string] $path);
+
+    Write-Host "##[info]Testing VSCode Extension inside $path..."    
+    Push-Location (Join-Path $PSScriptRoot $path)
+        npm run unittest
+    Pop-Location
+}
+
 Test-One '../QsCompiler.sln'
+Test-VSCode '../src/VSCodeExtension'
```

Comment:
To match the behavior the build.ps1 uses, you should skip executing VSCodeExtension tests if the environment variable is set to false.
```suggestion
if ($Env:ENABLE_VSIX -ne "false") {
    Test-VSCode '../src/VSCodeExtension'
} else {
    Write-Host "##vso[task.logissue type=warning;]VSIX building skipped due to ENABLE_VSIX variable."
}
```